src/Og.php

```diff
@@ -604,23 +606,6 @@ public static function groupManager() {
   }
 
   /**
-   * Get a role by the group's bundle and role name.
-   *
-   * @param string $entity_type_id
-   *   The group entity type ID.
-   * @param string $bundle
-   *   The group bundle name.
-   * @param string $role_name
-   *   The role name.
-   *
-   * @return \Drupal\og\OgRoleInterface|null
-   *   The OG role object, or NULL if a matching role was not found.
-   */
-  public static function getRole($entity_type_id, $bundle, $role_name) {
```

Comment:
As indicated in another comment getRole should still remain (although we can move it to OgRole itself ) as You may want to set the role's permissions [before](https://github.com/amitaibu/og/blob/8.x-1.x/tests/src/Functional/GroupSubscribeTest.php#L139-L143) any group as actually created

Also, just as a heads up, the entire OgRole will still undergo changes, once we'll have to introduce the concept of `OgRolePerBundle` and `OgRolePerGroup` (as we have in OG7)